pkgs/tools/archivers/bomutils/default.nix

```diff
@@ -13,7 +13,8 @@ stdenv.mkDerivation rec {
     sha256 = "1i7nhbq1fcbrjwfg64znz8p4l7662f7qz2l6xcvwd5z93dnmgmdr";
   };
 
-  makeFlags = [ "PREFIX=$(out)" ];
+  makeFlags = [ "PREFIX=$(out)" ]
+    ++ lib.optional stdenv.isDarwin "CXX=clang++";
```

Comment:
```suggestion
    ++ lib.optional stdenv.isDarwin "CXX=${stdenv.cc.targetPrefix}c++";
```

Reply:
Ah, makes sense. In that case, would it not be beneficial to drop the optional entirely, and just apply it everywhere?